config/routes.rb

```diff
@@ -30,7 +30,12 @@
       resources :users, only: :create do
         match :index, via: :delete, on: :collection, action: :destroy_by_email
       end
-      resources :mattermost, only: :create
+      resources :mattermost, only: [] do
```

Comment:
This is not a problem, but FYI you can also do it with `scope path: "mattermost"`